space/core/components/editor/rich-text-editor.tsx

```diff
@@ -33,7 +33,8 @@ export const RichTextEditor = forwardRef<EditorRefApi, RichTextEditorWrapperProp
         anchor: "",
       })}
       {...rest}
-      containerClassName={cn("relative pl-3 pb-3", containerClassName)}
+      containerClassName={cn("relative pl-3 pb-3 h-[100px]", containerClassName)}
+      editorClassName="h-[100px] border border-gray-100 rounded-md pl-3 pb-3 overflow-y-scroll"
```

Comment:
Why height is required at two places? Doesn't the parent auto expand? Wont this break the auto expand feature?

Reply:
No i have tested it out. I have added a scroll after a certain height. editor height needs to be given. I will remove the parent height though